rules/camunda-cloud/secrets.js

```diff
@@ -115,5 +115,5 @@ function isValidSecret(value) {
   return !value
     || !isString(value)
     || !value.includes('secrets.')
-    || /{{secrets\.[a-zA-Z0-9_]+}}/.test(value);
+    || /{{\s*secrets\.(\S+?\s*)}}/.test(value);
```

Comment:
```suggestion
    || /{{\s*secrets\.[\S]+\s*}}/.test(value);
```

This matches:

* Any amount of whitespace
* `secrets.`
* Any amount of characters, not including whitespace
* Any amount of whitespace

If we rather want to be on the safer side, we want to restrict to word only chars (cf. original rule) + `-`:

```suggestion
    || /{{\s*secrets\.[\w-]+\s*}}/.test(value);
```